conftest.py

```diff
@@ -104,3 +106,21 @@ def test_post_teardown():
 
     # Make sure we revert everything we might have changed to prefixers.
     amo.urlresolvers.clean_url_prefixes()
+
+
+@pytest.fixture
+def admin_group(db):
```

Comment:
Yeah, it's a bit disconcerting at first, it's what's called "pytest fixtures", it's an equivalent to unittest test fixtures (`setUp`, `tearDown`, `setUpClass`, `tearDownClass`, `_pre_setup`, `_post_teardown`,...), but by doing dependency injection.

It's VERY powerful, can reduce your tests to the minimum, without boilerplate, allows you to easily test your fixtures, and reuse them very easily everywhere, combine them, parameterize (parametrise? :) them...

At first it looks like black magic, the opposite of explicitness, but then you really get used to it, and can't imagine writing tests without that, trust me ;)

BTW: http://mathieu.agopian.info/presentations/2015_06_djangocon_europe/ and https://vimeo.com/channels/952478/134952909